components/about_build_modal/about_build_modal.test.jsx

```diff
@@ -129,6 +128,6 @@ describe('components/AboutBuildModal', () => {
             ...props,
         };
 
-        return shallow(<AboutBuildModal {...allProps}/>);
+        return shallowWithIntl(<AboutBuildModal {...allProps}/>);
```

Comment:
I don't see the need of the shallowWithIntl here, `AboutBuildModal` doesn't use intl in the shallow level.

Reply:
FormattedMessage requires it. If you use regular `shallow` it errors with:

```
 console.error tests/setup.js:55
    [React Intl] Could not find required `intl` object. <IntlProvider> needs to exist in the component ancestry. Using default message as fallback.

  console.error tests/setup.js:55
    The above error occurred in the <InjectIntl(FormattedMarkdownMessage)> component:
        in InjectIntl(FormattedMarkdownMessage) (created by AboutBuildModal)
```

Reply:
It doesn't happen in my machine. Works perfectly fine with shallow. Actually the error that you are pasting there is not related to the change, is about the `FormattedMarkdownMessage` so it shouldn't be related to this.

Reply:
👍